core/src/components/datetime/datetime-util.ts

```diff
@@ -242,12 +242,13 @@ export const parseDate = (val: string | undefined | null): DatetimeData | undefi
 };
 
 /**
- * Converts a valid UTC datetime string
- * To the user's local timezone
+ * Converts a valid UTC datetime string to JS Date time object.
+ * By default uses the users local timezone, but an optional
+ * timezone can be provided.
  * Note: This is not meant for time strings
  * such as "01:47"
  */
-export const getLocalDateTime = (dateString: any = ''): Date => {
+export const getDateTime = (dateString: any = '', timeZone: any = ''): Date => {
```

Comment:
```suggestion
export const getDateTime = (dateString: any = '', timeZone?: string): Date => {
```

Reply:
I only did the any because thats how `dateString` was done and tried to be consistent :) will update today